en/docs/GettingStarted/quick-start-guide.md

```diff
@@ -1,123 +1,182 @@
-
-    `http://www.mocky.io/v2/5185415ba171ea3a00704eed`
-
-    Copy this URL and use a new browser tab to navigate to it. You should see a JSON message saying `{"hello": "world"}` We are going to create an OAuth2.0 protected REST API for this service on API Manager.
-
-5. To create a new API from scratch let's click on the `Create API` button and chose the `Design a new REST API` option.
-  
+    __Note__  that this option is only available as long as there are no APIs created in the system.
+    
+    You can skip the following steps and jump to the next section, [subscribe to the API through the developer portal and generate keys](https://apim.docs.wso2.com/en/latest/GettingStarted/quick-start-guide/#creating-and-deploying-an-api-from-the-publisher-portal). 
+
+3. If you chose to __create an API from scratch__, let's use a mock REST service.
+ 
+    * Open [https://www.mocky.io/](https://www.mocky.io/) on your web browser. A mock service with a 
+        JSON response `{"hello": "world"}`  is provided by default on the landing page of the site. We 
+        will be using the service URL (`https://www.mocky.io/v2/5185415ba171ea3a00704eed`) which is shown 
+        in the mock service.
+
+     ![]({{base_path}}/assets/img/get_started/Mocky_io.png)
+         
+    * Optionally, to test this service, copy the service URL 
+        [https://www.mocky.io/v2/5185415ba171ea3a00704eed](https://www.mocky.io/v2/5185415ba171ea3a00704eed) on a new browser and navigate to it. You 
+        should see a JSON message saying 
+            
+            `{"hello": "world"}`. 
     
-    ![](../assets/img/get_started/design_new_rest_api.png)
+4. On the API Publisher Portal, click on the `Create New API` dropdown and choose the `Design a new REST API` option.
+   
+    ![]({{base_path}}/assets/img/get_started/design_new_rest_api.png)
 
 
-6.  Fill in the following details in the form that appears. Use the URL from mocky.io as the endpoint. Note to use the http protocol for now since using https for mocky.io requires importing its certificate into API Manager.
+5. Fill in the following details in the form that appears. Use the URL we obtained at step3 from mocky.io as the endpoint.
 
-    ![](../assets/img/get_started/api_create.png)  
+     
+    ![]({{base_path}}/assets/img/get_started/api_create.png)  
+         
+    !!! important
+        __Use the http protocol__ since using https for mocky.io requires importing its certificate  
+        into API Manager 
+        
+6. Click on the `Create & Publish` button once done. This will publish your first API on the developer portal as well as deploy it on the API gateway. You now have an OAuth2.0 secured REST API that is ready to be consumed.
```

Comment:
"Click on the `Create & Publish` button once done."

- When using steps in a sequence we do not need to add words such as "once done", "initially", "next", "finally" etc. so let's remove "once done".
- Reword this sentence as follows: Note that based on the Microsoft Manual of Style Guide we do not mention whether it is a "button", "dropdown" etc.
Click on `Create & Publish`."